package.json

```diff
@@ -55,16 +40,27 @@
   "private": true,
   "scripts": {
     "ava": "ava",
-    "build": "npm run clean && npm-run-all build:* && npm run build-function-templates",
+    "build": "yarn clean && yarn update:references && node scripts/test-all.js build",
```

Comment:
We are having all the functions as packages now, instead of just being folder at the root? Why is that? Nothing wrong with it, I was just curious. Also, since the structure changes, I suppose the current pipeline will fail to deploy these functions, right? Have you tried the new structure is okay with Azure, by manually deploying these, or were you testing the workers only with existing already-deployed functions? I am asking because in my experience, the functions have been really particular about folder structures.

Reply:
As we talk a few days ago, we are turning this in a monorepo, in this case, all functions except 'workers' are going to a folder, utils are going to another and then workers.

The deployment process is something that we need to figure out, but each "function" folder ('services' and 'worker') should contains everything necessary to be deployed in azure functions. We need to think about the utils, do we need to publish it in npm? can we use it without deploy it?

Reply:
About `utils` for now I'm referencing to the path: `"@online-service/utils": "file:../utils"` so we don't need to publish it. The only think is that we need to be sure that 'utils' is compiled before use it.